vm/src/stdlib/ssl.rs

```diff
@@ -743,19 +783,29 @@ fn convert_ssl_error(
     e: impl std::borrow::Borrow<ssl::Error>,
 ) -> PyBaseExceptionRef {
     let e = e.borrow();
-    match e.io_error() {
-        Some(io_err) => io_err.into_pyexception(vm),
-        None => match e.ssl_error() {
-            Some(e) => convert_openssl_error(vm, e.clone()),
-            None => vm.new_exception(
-                ssl_error(vm),
-                vec![
-                    vm.ctx.new_int(e.code().as_raw()),
-                    vm.ctx.new_str(e.to_string()),
-                ],
+    let (cls, msg) = match e.code() {
+        ssl::ErrorCode::WANT_READ => (
+            vm.class("_ssl", "SSLWantReadError"),
+            "The operation did not complete (read)",
+        ),
+        ssl::ErrorCode::WANT_WRITE => (
+            vm.class("_ssl", "SSLWantWriteError"),
+            "The operation did not complete (read)",
```

Comment:
is this message also `read` instead of `write`?

Reply:
Heh, nice catch